src/index.js

```diff
@@ -18,3 +18,27 @@ let game = new Game();
 game.startGame('player1', 'player2', 'player3');
 
 console.log('This is the JavaScript entry file - your code begins here.');
+// This is the base functionality for creating our alphabet buttons, as far as where 
+// they will appear in our files, I thought this would be the best place to keep it for now.
+let consonants = ['b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'v', 'w', 'x', 'y', 'z']
+let vowels = ['a', 'e', 'i', 'o', 'u'] 
+const alphabet = $(".alphabet")
+
+function consonantButtons() {
+    var consonantAppend = consonants.map(letter => {
+       $("<button/>"), {
+           text: letter, 
```

Comment:
We would love to understand what this is doing, so let's sit down and chat when you are feeling better.